src/components/TeamMemberTasks/TeamMemberTasks.jsx

```diff
@@ -599,18 +598,48 @@ const TeamMemberTasks = React.memo(props => {
                   }}
                   onClick={() => selectPeriod(days)}
                 >
-                  {days} days
+                  {days} {days === "1" ? "day" : "days"}
```

Comment:
What do you think about DRYing up this code block? We seem to use this line more than once (line 601 and 630).